formula-test/src/main/java/com/instacart/formula/test/TestFormulaObserver.kt

```diff
@@ -14,10 +16,12 @@ import kotlinx.coroutines.launch
 
 @OptIn(DelicateCoroutinesApi::class)
 class TestFormulaObserver<Input : Any, Output : Any, FormulaT : IFormula<Input, Output>>(
-    private val runtimeConfig: RuntimeConfig,
+    private val failOnError: Boolean,
```

Comment:
Adding a new `failOnError` flag that we can disable for tests where we expect failures.